tests/TestCase/TypeTest.php

```diff
@@ -568,6 +572,10 @@ public function testDeleteAll()
     public function testDeleteAllOnlySome()
     {
         $result = $this->type->deleteAll(['body' => 'cake']);
+        
+        //TODO : Not Sure
+        ConnectionManager::get('test')->getIndex()->refresh();
```

Comment:
Forcing a refresh seems reasonable given that we can't rely on elastic search merging the new segments quickly.